src/gui/src/pref_colors.cpp

```diff
@@ -0,0 +1,407 @@
+/*
+ * This file is part of the QPxTool project.
+ * Copyright (C) 2008-2012 Gennady "ShultZ" Kozlov <qpxtool@mail.ru>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file "COPYING" for the exact licensing terms.
+ * */
+
+#include <QAction>
+#include <QFrame>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QLineEdit>
+#include <QMenu>
+#include <QPushButton>
+#include <QSettings>
+#include <QVBoxLayout>
+
+#include "colorlabel.hpp"
+#include "pref_colors.hpp"
+#include "qpxsettings.hpp"
+#include <QDebug>
+
+prefColors::prefColors(QPxSettings *iset, QWidget *p, Qt::WindowFlags fl)
+    : QWidget(p, fl) {
+  QAction *tact;
```

Comment:
Codacy found an issue: [The scope of the variable 'tact' can be reduced.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)